src/node.rs

```diff
@@ -278,9 +265,31 @@ impl Taffy {
         Ok(self.nodes[node].is_dirty)
     }
 
-    /// Updates the stored layout of the provided `node` and its children
+    /// Updates the stored layout of the provided `node` and its children with measure
+    pub fn compute_measured_layout(
+        &mut self,
+        node: Node,
+        measure: &impl Measure,
+        size: Size<Option<f32>>,
+    ) -> Result<(), TaffyError> {
+        self.compute(node, size, measure);
+        Ok(())
+    }
+}
+
+struct NoopMeasure();
```

Comment:
this one can be probably made public

Reply:
Definitely agree. Although I'd probably lean towards `IdentityMeasure`?

Reply:
Totally